### PR TITLE
[Proto Annotations] Set default language package name

### PR DESCRIPTION
**Pull request by andreamlin**
Package names are configured in GAPIC config, but when no GAPIC config is provided, we should give a default package name.

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL

File: src/main/java/com/google/api/codegen/configgen/mergers/ProtoConfigMerger.java [hunk 1]
```diff
@@ -57,14 +56,10 @@ private ConfigMerger createMerger(Model model, String fileName) {
...
-      Interface apiInterface = model.getSymbolTable().lookupInterface(api.getName());
-      if (apiInterface != null) {
-        return apiInterface.getFile().getFullName();
-      }
+    String packageName = ProtoParser.getPackageName(model);
+    if (packageName != null) {
+      return packageName;
     }
-
     helper.error(model.getLocation(), "No interface found");
```
Comment (michaelbausor) [inline src/main/java/com/google/api/codegen/configgen/mergers/ProtoConfigMerger.java]: Should we update this message to something like "Failed to determine package name"?

Comment (andreamlin) [reply src/main/java/com/google/api/codegen/configgen/mergers/ProtoConfigMerger.java]: Done.

**Comment by andreamlin** [comment]
Yes, filed an issue https://github.com/googleapis/gapic-generator/issues/2299! Thanks for pointing that out.

**Pull request merged**